diff --git a/Makefile b/Makefile
index 781a220..30002b5 100644
--- a/Makefile
+++ b/Makefile
@@ -151,6 +151,7 @@ OMSimulator:
 	@echo
 	@$(MAKE) CC="$(CXX)" CXX="$(CXX)" OMTLMSimulator
 	@$(MAKE) OMSimulatorCore
+	install_name_tool -add_rpath $(BOOST_ROOT)/lib $(TOP_INSTALL_DIR)/bin/OMSimulator
 	test ! -z "$(DISABLE_RUN_OMSIMULATOR_VERSION)" || $(TOP_INSTALL_DIR)/bin/OMSimulator --version
 
 OMSimulatorCore:
